share/mrpt/config_files/rawlog-grabber/SICK_TIM_56x.ini

```diff
@@ -0,0 +1,53 @@
+# -------------------------------------------------------------------
+#  Config file for the `rawlog-grabber` MRPT application.
+#  Usage: 
+#      rawlog-grabber CONFIG_FILE.ini
+#
+#  Each section `[XXXXX]` but `[global]` defines a dedicated thread where a 
+#  sensor-specific driver runs. Each thread collects observations in parallel 
+#  and the main thread sort them by timestamp and dumps them to a RAWLOG file.
+#  The driver for each thread is set with the field `driver`, which must
+#  match the name of any of the classes in mrpt::hwdrivers implementing 
+#  a CGenericSensor.
+#
+# Read more online: 
+# http://www.mrpt.org/list-of-mrpt-apps/application-rawlog-grabber/
+# -------------------------------------------------------------------
+
+# =======================================================
+#  Section: Global settings to the application   
+# =======================================================
+[global]
+# The prefix can contain a relative or absolute path.
+# The final name will be <PREFIX>_date_time.rawlog
+rawlog_prefix		= ./dataset
+
+# Milliseconds between thread launches
+time_between_launches	= 300
+
+# Maximum time (seconds) between a sequence of observations 
+#  is splitted into sensory frames
+SF_max_time_span	= 0.005
+
+# Whether to use sensory-frames to group close-in-time observations (useful for some SLAM algorithms).
+use_sensoryframes 	= 0
+
+
+# =======================================================
+#  SENSOR: LASER1
+#   
+# =======================================================
+[LASER1]
+driver			= CSICKTim561Eth 
+
+sensorLabel  	= SICK_TIM_561
+pose_x			= 0		// Laser range scaner 3D position in the robot (meters)
+pose_y			= 0
+pose_z			= 0.31
+pose_yaw		= 0		// Angles in degrees
+pose_pitch		= 0
+pose_roll		= 0
+
+m_ip = 192.168.0.1  // Default IP address
+m_port = 2111       // Default IP Port
+m_process_rate = 15 // Default Herzt
```

Comment:
Thanks for the PR! :+1: 

Just a comment on these `m_*` variable names: for convention and homogeneity with the rest of driver classes, variables in configuration files don't use to have that prefix. Please, take a look at other classes, and update the PR if you see it easy to do....

Reply:
Thanks for the comment. The `ini` file has been updated without prefix, and the PR has been updated. What else can I do for this PR? Please feel free to leave a message at here, I am so glad to do it. Thanks!